src/Workspaces/SharedUtilitiesAndExtensions/Compiler/Core/Options/PerLanguageOption2.cs

```diff
@@ -61,6 +65,20 @@ private void VerifyNamingConvention()
             Debug.Assert(!Definition.ConfigName.StartsWith(OptionDefinition.VisualBasicConfigNamePrefix, StringComparison.Ordinal));
         }
 
+#if !CODE_STYLE
```

Comment:
Code using IGlobalOptionService shouldn't be in this layer. Why is it needed here and not simply in LSP layer?